### DISCUSSION TITLE
Is there a way to use only the subdomain multi-tenancy?

### DISCUSSION THREAD

**Discussion by VaZark** [django-tenants/django-tenants#753] [Q&A] [answered]
I plan on using separate databases for each tenant (with common schema), so using the built-in support for multiple databases with separate roles for schema seems to be a good idea. 

However, I want to know if it is possible to leverage just the subdomain feature so that I can use the subdomain as the key to identify the database to use with `using().`

**Comment by tomturner** [comment, marked as answer]
No not really the key is the title of the project.

Tom
